lib/Pages/Handler/Modify.php

```diff
@@ -91,24 +93,38 @@ public function initialize(Zikula_Form_View $view)
      */
     public function handleCommand(Zikula_Form_View $view, &$args)
     {
+        if ($this->_page->get()) {
+            // DO NOT REMOVE!! This is important to be called before ->getValues() below.
```

Comment:
You're right, this wasn't the smartest thing I've ever done, see https://github.com/zikula-modules/Pages/commit/f01591afd7d6be09a19d230d6d20e3c548bfe6ad :boat: